test_scripts/Defects/6_1/2460_Resuming_SubscribeWayPoints.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/2460
+-- Description: SDL should send SubscribeWayPoints request to HMI during resumption
+--
+-- Preconditions:
+-- 1) App is registered
+-- 2) PTU is performed
+-- 3) App is activated
+-- 2) App is subscribed to WayPoints
+--
+-- Step:
+-- 1) Apps reconnects with actual hashId
+-- SDL does:
+-- a. perform resumption of data and HMI level after transport reconnect
+-- b. transfer SubscribeWayPoints request to HMI during app resumption
+-- c. not transfer SubscribeWayPoints response to the app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local hashId
+
+--[[ Local Functions ]]
+local function ptUpdateForApp(pTbl)
+  pTbl.policy_table.app_policies[common.getConfigAppParams(1).fullAppID].groups = { "Base-4", "WayPoints" }
+end
+
+local function sendSubscribeWaypoints()
+  local cid = common.getMobileSession():SendRPC("SubscribeWayPoints",{})
+  common.getHMIConnection():ExpectRequest("Navigation.SubscribeWayPoints")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  common.getMobileSession():ExpectNotification("OnHashChange")
+  :Do(function(_,data)
+      hashId = data.payload.hashID
+    end)
+end
+
+local function rai()
+  common.getMobileSession():StartService(7)
+  :Do(function()
+      common.app.getParams().hashID = hashId
```

Comment:
```suggestion
      common.getConfigAppParams().hashID = hashId
```

Reply:
`getConfigAppParams` this is an old function that is used for old scripts, in case function is not defined as `getConfigAppParams`, the `common.app.getParams` should be used